rllib/agents/dqn/simple_q.py

```diff
@@ -85,12 +88,19 @@
     # Size of the replay buffer. Note that if async_updates is set, then
     # each worker will have a replay buffer of this size.
     "buffer_size": DEPRECATED_VALUE,
-    # Deprecated for Simple Q because of new ReplayBuffer API
-    # Use MultiAgentPrioritizedReplayBuffer for prioritization.
+    # The following values have moved because of the new ReplayBuffer API
     "prioritized_replay": DEPRECATED_VALUE,
+    "learning_starts": DEPRECATED_VALUE,
+    "replay_batch_size": DEPRECATED_VALUE,
+    "replay_sequence_length": DEPRECATED_VALUE,
+    "prioritized_replay_alpha": DEPRECATED_VALUE,
+    "prioritized_replay_beta": DEPRECATED_VALUE,
+    "prioritized_replay_eps": DEPRECATED_VALUE,
     "replay_buffer_config": {
         # Use the new ReplayBuffer API here
         "_enable_replay_buffer_api": True,
+        # How many steps of the model to sample before learning starts.
+        "learning_starts": 1000,
```

Comment:
IMO timesteps are a better unit to measure this with, rather than the number model forward passes.

Reply:
I'll specify this!
Thanks for all your awesome comments!
Really cool stuff 💯

Reply:
Just realized that this phrase is all over the library.